### ISSUE TITLE
Post SDPS as an option

### ISSUE THREAD

**Issue by Strumph-8** [kauffman12/EQLogParser#190]
When you send a parse to EQ it only does DPS. Is there a option I haven't found to send SDPS or could there be?

**Comment by kauffman12** [comment]
People get confused by DPS vs SDPS which is why it only does the one. Especially if it were configurable and some people showed one and some people showed the other.

Total damage is the important number to see who contributed the most and DPS is useful to see what people are capable of outside of death or other downtime during a fight. Not sure if it's worth changing at this point as it's been like this for years.

**Comment by kauffman12** [comment]
I'll leave this open for now. If more people really want it I could add an option for it. I do think it could cause confusion tho.

**Comment by LamahHerder** [comment]
DPS is a good gauge of character damage contribution
SDPS is a good gauge of player damage contribution

Everyone is used to DPS thou so it would add confusion

Why is JoeBobSuperDPS so low on the dps ranks?!
Because they engaged when the boss was at 70% health and only attacked half the mobs the rest of the raid killed...

**Comment by Strumph-8** [comment]
> People get confused by DPS vs SDPS
Yes I'd say a lot of people in my guild are. They look at the DPS and go OOooooohhhh JoeBob did GREAT! Though as Lamah mentioned JoeBob was asleep and only attacked at 70%.

So if I wanted to link a parse to compare one class I'd want to do SDPS to point out poor performers, so the Class Leader or myself can work. 

The parse shows deaths when you link it, so if anybody were to question JoeBob's contribution it's there. Though most people I know don't pay attention to anything more than DPS and possibly DMG.

I used to use SDPS with the other parser, but only use yours now and love it. Just thought I'd request this option, but sounds like I may be the only one.
